pgpu-render/pgpu.h

```diff
@@ -0,0 +1,140 @@
+#include <cstdarg>
```

Comment:
I think this file is auto-generated? It's actually a bit hard to tell as some of the comments look hand-written. In any case, would it be possible to add a line at or near the top explaining what's going on?

Reply:
This is auto-generated. I'll configure the generator to add a comment at the top marking it as such.